coldfront/core/portal/views.py

```diff
@@ -52,6 +57,7 @@ def home(request):
         context['savio_projects'] = savio_projects
         context['vector_projects'] = vector_projects
         context['allocation_list'] = allocation_list
+        context['cluster_username'] = request.user.userprofile.cluster_uid
```

Comment:
`cluster_uid` is the Linux UID, not the username. The username is stored directly in `User.username`.

The username can either be your email address (if you don't have a cluster account) or your username (if you do).

To check if someone has cluster access under at least one project, you can reuse the results of the query I mentioned in the previous comment. If this is `True`, you can display the username.

(The name of the attribute, 'Cluster Account Status' is a little misleading--it should be named 'Cluster Access Status', but I'll fix that later.)